check_file_sizes.sh

```
@@ -0,0 +1,250 @@
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Настройки по умолчанию
+EXTENSIONS="go"
+EXCLUDE_TESTS=true
+
+# Функция для получения оценки файла
+get_file_rating() {
+    local lines=$1
+    local filename=$2
```

Shellcheck (0.11.0)

[warning] filename appears unused. Verify use (or export if used externally).

(SC2034)